src/analyzer/analyzerbeats.h

```diff
@@ -46,7 +48,6 @@ class AnalyzerBeats : public Analyzer {
     bool m_bPreferencesFastAnalysis;
 
     mixxx::audio::SampleRate m_sampleRate;
-    SINT m_totalSamples;
-    int m_iMaxSamplesToProcess;
-    int m_iCurrentSample;
+    SINT m_maxFramesToProcess;
+    SINT m_currentFrame;
```

Comment:
Can't we use FramePos here?

Reply:
FramePos is not available in the 2.3 branch.